src/Kdump.ycp

```diff
@@ -940,6 +950,30 @@ boolean addCrashkernelToKexec (string crashkernel_value)
 }
 
 /**
+ * Update xen_append
+ *
+ * It relies on section_pos.
+ * It refuses to update the parameter unless is_xen().
+ *
+ * @return boolean true if was updated.
+ */
+boolean UpdateXenAppendParameter (string key, string value) {
+	if (!is_xen()) {
+		y2milestone("Not Xen: refusing to update Xen boot section");
+		return false;
+	} else {
+		string xen_append = BootCommon::sections[section_pos, "xen_append"]:"";
+		BootCommon::sections[section_pos, "xen_append"] =
+			BootCommon::setKernelParamToLine(xen_append, key, value);
+		// added flag which means that section was changed bnc #432651
+		BootCommon::sections[section_pos, "__changed"] =
+			(BootCommon::sections[section_pos, "xen_append"]:"" != xen_append);
+		y2milestone("Write boot section to Xen boot section %1", BootCommon::sections[section_pos]:nil);
```

Comment:
NP: The message seems a bit confusing to me, a simple `Using Xen boot section: %1` would be better IMHO.